components/formats-gpl/matlab/createMinimalOMEXMLMetadata.m

```diff
@@ -58,7 +58,7 @@
 else
     java_true = java.lang.Boolean.TRUE;
 end
-metadata.setPixelsBinDataBigEndian(java_true, 0, 0);
+metadata.setPixelsBigEndian(java_true, 0, 0);
```

Comment:
Extra `,0` should be removed - see https://ci.openmicroscopy.org/view/Failing/job/BIOFORMATS-DEV-breaking-matlab/442/testReport/